db/controllers.js

```diff
@@ -52,3 +53,14 @@ exports.postComment = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.patchReview = (request, response, next) => {
+  const body = request.body;
```

Comment:
Could take the id off the body object before sending through, to save you sending the whole body to the model 🕵️